cmd/configure.ac

```diff
@@ -257,5 +257,22 @@ fi
 fi
 AM_CONDITIONAL([USE_INTERNAL_BPF_HEADERS], [test "x$host_bpf_headers_usable" = "xno"])
 
+AC_DEFINE([MISSING_FIELD_INITIALIZERS_USABLE], [1], [-Wmissing-field-initializers are usable])
+AC_MSG_CHECKING([whether MISSING_FIELD_INITIALIZER warnings are usable])
+AC_COMPILE_IFELSE([
+#include <linux/resource.h>
+#pragma GCC diagnostic error "-Wmissing-field-initializers"
+struct sc_mount{const char *path;int foo;};
+void foo() { struct sc_mount mounts[[]] = { {.path = "x"}, {}, }; }
```

Comment:
I think if the string was double quoted (`[[ ... ]]`), then you would not need to  quote square brackets.